### ISSUE TITLE
Use of isdbg 

### ISSUE THREAD

**Issue by spgennard** [RechInformatica/rech-cobol-debugger#1]
The README says:

```text
Configuration steps

It is suggested to use isCOBOL™ command-line debugger isdbg since
the commands fired by this extension respect the commands expected 
by isdbg.
```
As the documentation only suggests the use of isdb, can you suggest what other command-line 
debuggers can be used?   for example gdb, adb, cdb?   

Some of the above debuggers can handle COBOL.

Or is it just the documentation should say this is a requirement rather than a suggestion,
if so, could the name of the extension include something to reflect it is for use with
isCOBOL Evolve only.

for example:

"Debug COBOL files with Visual Studio Code"

to

"Debug isCOBOL Evolve COBOL files with Visual Studio Code

Then you can be proud & explicit what is for!

(Please include the versions of iSCOBOL it works with please)

**Comment by GustavoASC** [comment]
Hello @spgennard, how are you?

The goal of this extension is to provide a general-purpose debug adapter for any COBOL compiler. The only requirement is to have a command-line debugger which responds to the same commands provided by _isdbg_. Since we use isCOBOL compiler at the company,  we only have access to this debugging tool right now, so I don't know how to answer your question and tell you whether gdb, adb and cdb would work for the moment.

We plan to test/validate different command-line debuggers with this extension on the next months.

For these reasons, we won't rename this extension to 'Debug isCOBOL Evolve COBOL files with Visual Studio Code' or something like that, since the goal is make it a general-purpose debug adapter.

I am updating the README.md to make it clearer.

Hope it helps!
Thanks in advance :+1:

**Comment by spgennard** [comment]
Hi to you too,

Myself and my family are okay


Okay, I understand what you are saying and the purpose of the debugger
extension.

Do you have a specification for the 'generic' debugger?

Then I can see how close it the gdb interface that COBOL-IT and GnuCobol
use fits into it.

I might be able throw together and adapter application.

**Comment by GustavoASC** [comment]
Hello,
We'll check whether a good and documented specification already exists and we'll keep in touch with you on the next days, telling you what we find out.

Thanks and have a good weekend! 😄

**Comment by GustavoASC** [comment]
Hello @spgennard,

We've just released version 1.0.34, which supports loading command names and regular expressions directly from an external configuration file. This way you can 'tell' the extension the commands your external debug process provides and specify regular expressions to detect whether the expected output has been matched.

Additional information is located on README.

I'm closing this issue because I believe we can use this external config file from now on.
If you have any doubt/problem/suggestion please open new issues so we can handle the specific situations.

Thanks and have a great week! 😃
